### PR TITLE
feat: added search component

### PR DESCRIPTION
**Pull request by Ayoazeez26**
resolves: #9

### CODE REVIEW

**Comment by stephanniegb** [comment]
Screenshot 2024-10-03 at 7 24 20 AM
Deployment fails

**Comment by Ayoazeez26** [comment]
@stephanniegb fixed

File: pnpm-lock.yaml
Comment (stephanniegb) [inline pnpm-lock.yaml:1]: Why do we have this file?

Comment (Ayoazeez26) [reply pnpm-lock.yaml:1]: That was what made the deployment fail, because I added a new dependency, you'll see that in the package.json file

Comment (Ayoazeez26) [reply pnpm-lock.yaml:1]: Because I saw a pnpm-lock.yaml file in the project directory, then I assumed it was okay to use it

Comment (stephanniegb) [reply pnpm-lock.yaml:1]: Oh. Well just use npm

Comment (Ayoazeez26) [reply pnpm-lock.yaml:1]: Alright, i'll do that now

**Comment by Oshioke-Salaki** [comment]
Looks good to me🚀

**Comment by Oshioke-Salaki** [comment]
lgtm 🚀. Thank you for your contribution

**Pull request merged**